Update AboutUs, PPP, and UG (#31)
AY2324S1-CS2103T-T13-4/tp: docs/UserGuide.md

Oops! There seems to be an error, please check the format of your command again.
```

### Editing a fosterer's detail : `edit`

Edits the details of a fosterer stored in the address book.

**Format** 1: `edit INDEX`

**Parameter**: `INDEX`

* Index of a fosterer to be edited is shown in the list obtained by  the find/list command.

**Format** 2: (In the fosterer profile view) `edit`

**Examples**:

> *  `list` followed by `edit 3` Generates the edit-window for the 3rd fosterer in the address book

Reviewer: ```suggestion
*  `list` followed by `edit 3` generates the edit-window for the 3rd fosterer in the address book
```

*  `list` followed by `view 3` followed by `edit` allows making changes from the fosterer profile view. 

**How to Edit**:
1. Type in `list` command to see the list of fosterers in the address book.
2. Type `view INDEX` to enter the profile page of the fosterer with the corresponding `INDEX` from the `list` command. 

> 3. Type `edit` to edit the fosterer's details (alternatively, you can skip step 2 by typing `edit INDEX` command.)

Reviewer: Think it makes sense for `view` to be an alias of `edit` (same functionality)?

4. In the command line, type the name of the field you want to edit and press enter. This converts a plain text into a 

> text field with the detail. 

Reviewer: ```suggestion
4. Type in the command line to search the list of fields, and press enter to edit the first match. This creates a text field around the detail corresponding to the chosen field.
```

> 5. Edit the content of the detail and press enter. 

Reviewer: ```suggestion
5. Edit the content of the detail and press enter to confirm the changes, or the esc key to cancel them. 
```

6. Repeat step 4-5 until you made all your edits. 

> 7. Type in `save` command to save the changes and prompt you back to the home window.

Reviewer: ```suggestion
7. While the command line is empty, press enter to save all applied changes to all fields, or the esc key to revert them to before the edit command. Both will send you back to the home window.
```
